docs/source/user_guide/backends/omnisci.rst

```diff
@@ -124,11 +127,12 @@ is raised.
 `MapDCursor` class was created just to allow `ibis.client.Query.execute`
```

Comment:
I would remove from L124 to L132 (https://github.com/ibis-project/ibis/pull/2265/files#diff-4ae26e229bda1f1f9f4fda583d639683L124-L132)

and I would add something like:

The `execution` method, under the hood, will use some `cursor` class that will convert the result from the database to some data frame format (e.g. Pandas dataframe, GeoPandas dataframe, cuDF dataframe).